src/Symfony/Component/RateLimiter/Limit.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+/**
+ * @author Valentin Silvestre <vsilvestre.pro@gmail.com>
+ *
+ * @experimental in 5.2
+ */
+final class Limit implements LimitInterface
+{
+    private $availableTokens;
+    private $retryAfter;
+    private $accepted;
+
+    public function __construct(int $availableTokens, \DateTimeImmutable $retryAfter, bool $accepted)
```

Comment:
Can `$accepted` be inferred from `$availableTokens`? (`$availableTokens < 0`)?

Reply:
You can have 2 available tokens and ask for 1 or 5 :) So it really depend I think

Reply:
ah, right you are!